x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/analytics/action/PostAnalyticsEventAction.java

```diff
@@ -143,6 +144,10 @@ public boolean isDebug() {
             return debug;
         }
 
+        public Map<String, List<String>> headers() {
```

Comment:
this is not used, right?

Reply:
good catch, it was leftover from before I found out that I had to check the canonical xcontent type. I will remove 👍